plugins/calendar/pkg/calendarImp.go

```diff
@@ -0,0 +1,140 @@
+package pkg
+
+import (
+	"bytes"
+	"encoding/csv"
+	"errors"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+const (
+	dbFile         = "calendar_db.csv"
+	dbColumns      = "date,title,description\n"
+	dbColumnsTitle = "Date\tTitle\tDesc."
+	dbFileMode     = 0755
+)
+
+type Calendar interface {
+	Add(date time.Time, title, description string) (string, error)
+	List() (string, error)
+	Remove(title string) (string, error)
+}
+
```

Comment:
If your package is heavily dependent on one struct, like this one, it's a good practice to create a constructor to it.

So you can create a func like this:
```
func New() &Calendar {
    return &Calendar{}
}
```

With this other packages can use this one like `calendar.New()` which is nice.